caterva2/services/sub.py

```diff
@@ -1160,9 +832,10 @@ def concatstackhelper(payload: models.ConcatStackPayload, user: db.User = Depend
     srcs, dst = payload.srcs, payload.dst
     # src should start with a special root or known root
     for src in srcs:
-        if not src.startswith(("@personal", "@shared", "@public")) and src not in settings.database.roots:
+        if not src.startswith(("@personal", "@shared", "@public")):
```

Comment:
AFAIK, there is no support for other roots than these, so I wonder if this check is necessary.

Reply:
In this specific case if the src parameter has an unexpected value we send a "400 Bad Request" response, so the check is still relevant.
But there are many other places that still need clean-up, either in this PR or in a follow up. For example just below for the dst argument, there is still code for supporting the old roots, this should be change to instead return an error.